### PR TITLE
Add getmail6

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/getmail6/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "getmail6" %}
+{% set version = "6.18.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive//v{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/getmail6/meta.yaml:9]:
```suggestion
  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
```

File: recipes/getmail6/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  sha256: 1dda163d20d7c67571d66234d1749e2209bca8cbcb7b944a2381837a82b88661
+
+build:
+  noarch: python
```
Comment (ocefpaf) [inline recipes/getmail6/meta.yaml:13]: Looks like it uses scripts and cannot be noarch: https://github.com/getmail6/getmail6/blob/master/setup.py#L84

File: recipes/getmail6/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+    - getmail --help
+    - getmail_fetch --help
+    - getmail_maildir --help
+    - getmail_mbox --help
+    - getmail-gmail-xoauth-tokens --help
```
Comment (ocefpaf) [inline recipes/getmail6/meta.yaml:32]: You'll may need entry_points for these, not sure how upstream define these.

Comment (jan-janssen) [reply recipes/getmail6/meta.yaml:32]: The issue is they just copy the shell scripts https://github.com/getmail6/getmail6/blob/master/getmail_fetch

Comment (ocefpaf) [reply recipes/getmail6/meta.yaml:32]: Maybe skip Windows then?

Comment (jan-janssen) [reply recipes/getmail6/meta.yaml:33]: Ok, done

**Pull request merged**